src/server/debug.rs

```diff
@@ -882,6 +882,14 @@ impl<E: Engine> Debugger<E> {
             "middle_key_by_approximate_size".to_string(),
             escape(&middle_key),
         ));
+        res.push((
```

Comment:
would this be a super large string? If so, Is there a way to add an option for printing it?